.gitattributes

```diff
@@ -0,0 +1,21 @@
+# Package manager lock files
+Pipfile.lock linguist-generated
+package-lock.json linguist-generated
```

Comment:
Could we also add `-diff` and `-merge` for these? It rarely makes sense to resolve these by hand.